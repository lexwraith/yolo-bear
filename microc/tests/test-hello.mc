main()
{
  print('i');
  print(71);
  print(1);
}
