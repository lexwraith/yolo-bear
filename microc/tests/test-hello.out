i
71
1
